### ISSUE TITLE
ItChat's MIT license is not recognised

### ISSUE THREAD

**Issue by danielcompton** [licensee/licensee#462]
### Describe the bug

https://github.com/littlecodersh/ItChat uses the MIT license, but licensee doesn't detect their [license correctly](https://github.com/littlecodersh/ItChat/blob/master/LICENSE).

### Steps to reproduce the behavior

```
$ licensee detect littlecodersh/ItChat --remote
License:        NOASSERTION
Matched files:  LICENSE
LICENSE:
  Content hash:  fe946012c87c55c00c38b4f060dffa862111f467
  License:       NOASSERTION
  Closest non-matching licenses:
    MIT similarity:      92.93%
    NCSA similarity:     71.60%
    BSL-1.0 similarity:  63.16%
```

### Expected behavior

ItChat's license is detected as MIT.

### Additional context

Diffing this seems pretty similar, the main differences are line breaks and the header

```diff
-MIT License
-
-Copyright (c) [year] [fullname]
+**The MIT License (MIT)**
+
+Copyright (c) 2017 LittleCoder ([littlecodersh@Github](https://github.com/littlecodersh))

**Comment by mlinksva** [comment]
Same problem as #435 (markdown normalization regression), fixed by #450.

FWIW licensee can tell you what it thinks the diff is (after normalization). Licensee 9.14.0:

```
$ bundle exec licensee diff https://github.com/littlecodersh/ItChat --license=mit
Comparing to MIT License:
Input Length:      1096
License length:    1020
Similarity:      92.93%
diff --git a/LICENSE b/LICENSE
index 0ce42af..31e4ce0 100644
--- a/LICENSE
+++ b/LICENSE
@@ -1,3 +1,4 @@
{+the mit license (mit) copyright (c) 2017 littlecoder (littlecodersh@github)+}
permission is hereby granted, free of charge, to any person obtaining a copy of
this software and associated documentation files (the "software"), to deal in
the software without restriction, including without limitation the rights to
```

HEAD:

```
$ bundle exec licensee diff https://github.com/littlecodersh/ItChat --license=mit
Comparing to MIT License:
Input Length:       1020
License length:     1020
Similarity:      100.00%
Exact match!
```

**Comment by danielcompton** [comment]
Thanks for the quick response. Sorry about the duplicate, I should have checked this running from source, not the latest released version.
